src/lib/Liblog/chk_file_sec.c

```diff
@@ -472,5 +473,6 @@ tmp_file_sec(char *path, int isdir, int sticky, int disallow, int fullpath)
 		}
 	}
 
+	free(real)
```

Comment:
This comment is for line 459, but I couldn't comment there. Please check the return value of malloc() against NULL and not zero.

Reply:
Need a semicolon at the end of this line.